cmdline/bin/dcat.dart

```diff
@@ -36,34 +36,36 @@ void main(List<String> arguments) {
   dcat(paths, argResults[LINE_NUMBER]);
 }
 
-Future dcat(List<String> paths, bool showLineNumbers) {
+Future dcat(List<String> paths, bool showLineNumbers) async {
   if (paths.isEmpty) {
     // No files provided as arguments. Read from stdin and print each line.
-    return stdin.pipe(stdout);
+    stdin.pipe(stdout);
   } else {
-    return Future.forEach(paths, (path) {
+    for (var path in paths) {
       int lineNumber = 1;
-      Stream<List<int>> stream = new File(path).openRead();
-      return stream
+      Stream stream = new File(path)
```

Comment:
I'd call this `lines`. I care more about what it's a stream *of* than I do that it's a stream.